recipe/meta.yaml

```
@@ -283,16 +316,26 @@ outputs:
         - pip
         - expecttest
         - xmlrunner
+        # Required by run_test.py
+        - pytest-rerunfailures
+        - pytest-shard
+        - pytest-flakefinder
+        - pytest-xdist
       imports:
         - torch  # [not (aarch64 and cuda_compiler_version != "None")]
```

This should at least be
```suggestion
      imports:
        - torch  # [(build_platform == target_platform) and (cuda_compiler_version == "None")]
```
Though IMO we should at least be able to run the import tests in emulation? Then it'd be
```suggestion
      imports:
        - torch  # [cuda_compiler_version == "None"]
```

Similar clean-ups would be beneficial below